### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by saddy001** [mozilla/voice-web#1104]
Suggestions for https://voice.mozilla.org/de:
* The recordings should be normalized (at least for playback) because they differ vastly in volume
* After clicking yes/no the next recording should automatically be played to incite the user to validate even more samples

**Comment by jbinard** [comment]
I agree. I validated a lot of records and it's a pain to always switch the volume between each record.

**Comment by jf99** [comment]
The first one is a duplicate of #336 and #445.

The second one should be optional and is a duplicate of #337.
